Library/ViewModels/ShareViewModel.swift

```diff
@@ -42,17 +41,11 @@ public protocol ShareViewModelInputs {
   /// Call from the UIActivityViewController's completion handler.
   func shareActivityCompletion(with data: ShareActivityCompletionData)
 
-  /// Call from the SLComposeViewController's completion handler
-  func shareComposeCompletion(result: SLComposeViewControllerResult)
-
   /// Call when the direct-share twitter button is pressed.
   func twitterButtonTapped()
 }
 
 public protocol ShareViewModelOutputs {
-  /// Emits when the share compose controller show be presented.
-  var showShareCompose: Signal<SLComposeViewController, NoError> { get }
```

Comment:
These outputs were not being used so I think all of the code referring to the old `Social` framework was unused.